src/main/scala/com/crealytics/spark/v2/excel/ExcelHelper.scala

```diff
@@ -135,6 +135,7 @@ class ExcelHelper private (options: ExcelOptions) {
   def getRows(conf: Configuration, uri: URI): Iterator[Vector[Cell]] = {
     val workbook = getWorkbook(conf, uri)
     val excelReader = DataLocator(options)
+    // todo this does not work with streaming reader
```

Comment:
are in you a position to continue with the 'todo'

Reply:
not yet, I removed the TODO for now...